### DISCUSSION TITLE
Does the rt module have refraction?

### DISCUSSION THREAD

**Discussion by lysfeng111** [NVlabs/sionna#637] [Q&A] [answered]
Does the rt module have refraction?

**Comment by faycalaa** [comment]
Hi @lysfeng111 ,

Sionna RT does not currently support refraction.

**Comment by lysfeng111** [comment]
Will there be refraction later?

**Comment by faycalaa** [comment, marked as answer]
As indicated in #528, we have an ongoing effort targeting refraction.

**Reply by redcoatwright** [reply]
Hi, this may be a dumb question but does this mean that Sionna RT can't simulate transmission through objects/materials on a scene? 

I'm assuming because refraction would be an inherent part of that but wondering if there's perhaps just a naive implementation where it assumes the boundary is planar or has no refraction properties.

**Comment by lysfeng111** [comment]
Thank you so much. I'm Looking forward to this feature.
